### DISCUSSION TITLE
KusionStack 第1次周会(中国区) - 2022-06-09

### DISCUSSION THREAD

**Discussion by chai2010** [KusionStack/community#6] [Meeting]
## 时间地点形式

- 时间：北京时间每周四晚上8点
- 会议：加入 [钉钉群（群号 42753001）](https://h5.dingtalk.com/circle/healthCheckin.html?cbdbhh=qwertyuiop&dtaction=os&4ebe6=d3d59&corpId=ding707e3346ab188ded2040d0a6270bd858)，钉钉视频会议
- 周期：开源周会和技术分享交替进行，如果没有分享则跳过

## 前一个周会任务回顾

- 上次周会 Action 回顾（第1次跳过）
- KusionStack Open Day 开源直播
- 代码仓库和中文文档上线
- 基于 github 的工作流初步配置完成

## 本次会议议程（可讨论补充）

- 开源周会的形式/时间/周期/软件平台
- 开源周会相关任务的管理收集到community项目的白板？
- 新用户使用安装的文档和教程的整套流程问题收集和验证
- 外部同学如何参与开发的文档和流程规范
  - 开源分支名称规范
  - 代码评审团队配置 
  - 完整的贡献流程文档（安装使用->如果提问题->找遗留问题->克隆代码->本地构建测试->提交PR->评审和CLA->完成）
- 同步英文文档、便于全球用户参与
- 测试覆盖率、代码规范
- Windows平台支持

## 下周技术分享主题（没有可以跳过）
- Kusion 在云原生SRE实践
- KCL 语言设计和实现

## 其他内容

[a comment by an automated account is not shown]

**Comment by chai2010** [comment]
# 会议记录

- 会议公告: https://github.com/KusionStack/community/discussions/6
- 会议时长: 晚上 8:00 - 9:00

## 议题议程

### 1. 新同学自我介绍

首先参会同学做了自我简介，共有来自蚂蚁、快手科技、北京东方康程通信和阿里巴巴 4 个公司的 10 位学参与了本次开源周会，感谢大家参与 🙏


| Name   | Company  | GitHub
| -----  | ----  | ----  |
| 柴树杉  | 蚂蚁集团 | @chai2010 |
| 杨英明  | 蚂蚁集团 | @elliotxx |
| 徐鹏飞  | 蚂蚁集团 | @Peefy |
| 袁昊  | 蚂蚁集团 | @howieyuen |
| 李波波  | 蚂蚁集团 | @markliby |
| 张正  | 蚂蚁集团 | @He1pa |
| 宗喆  | 蚂蚁集团 | @zong-zhe |
| 邓慧  | 快手科技 |  @freez-dh |
| 张同来  | 北京东方康程通信 | @ztl214096602 |
| 周涛 | 阿里巴巴 | @iamzhout |

### 2. KusionStack 一周回顾

来自蚂蚁的 @chai2010 简要介绍 KusionStack 开源 Open Day 和开源的仓库和文档等。

- 项目主页: https://kusionstack.io
- Github 仓库: https://github.com/KusionStack
- KusionStack Open Day: https://kusionstack.io/blog/2022-open-day

### 3. 开源周会形式讨论

开源项目最有价值的是开源社区、是参与开源活动的同学，如何为 KusionStack 社区同学提供一个方便自由讨论技术的平台和环境是大家最关注的问题。第一次开源周会是周四晚上 8:00- 8:45 开会（节假日时间顺延一周），同时结合 KusionStack 相关的技术分享交流，时长可以跟进具体情况变化酌情调整。

同时来自快手科技的 @邓慧 也给出了补充建议：视频会议属于同步沟通、虽然便于大家交流和即时反馈，但是不利于复杂问题的深度沟通，也不利于交流成果的进一步沉淀，有些同学也可能因为时间冲突而不能每场都参与。因此建议，KusionStack 能够最大化发挥 Github 的 Issue 等异步的沟通机制作为开源只会的补充形式。

来自北京东方康程通信的 @ztl214096602 同学也分享了对开源社区治理的经验和看法：对于经常参与社区治理、开发等活动的参与度比较高的同学作为开源决策参为维护者是有益的，同时核心参与同学可以主动跟进外围的新同学的问题和提供辅助有助于社区成长。同时也希望 KusionStack 的博客能接受更多用户的案例和技术分享投稿。

结论：作为试水阶段，开源周会是周四晚上 8:00- 8:45 开会（节假日时间顺延一周），同时结合 KusionStack 相关的技术分享交流，时长可以跟进具体情况变化酌情调整。同时希望能参与组织周会的同学积极主动报名自荐。

### 4. 高优先级的问题

- 新用户的使用安装文档问题收集，希望大家提供 Issue 积极反馈问题
- 外部同学如何参与开发的文档和流程规范
- 同步英文文档、便于全球用户参与
- 测试覆盖率、代码规范
- Windows平台支持

**Reply by lizongyu0912** [reply]
👍

**Comment by chai2010** [comment]
视频回放：https://space.bilibili.com/228717294/channel/seriesdetail?sid=2397933
